dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -136,4 +136,13 @@ extraProperties.name=Additional properties
 extraProperties.description=Additional properties related to session management.
 
 modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
-modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
\ No newline at end of file
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
+
+allow.serialized.access.name=Allow serialized access
+allow.serialized.access.description=Enable this option to allow serializing access to session data.
+
+max.wait.time.name=Max wait time
+max.wait.time.description=The amount of time, in milliseconds, a servlet waits on a session before it continues execution.
+
+access.on.timeout.name=Access on timeout
+access.on.timeout.description=This property gives the servlet access to the session, which allows normal execution even if the session is still locked by another servlet. Default is true, set to false to make the servlet execution abort when the session requests a time out.
```

Comment:
`...Default is true, set to false to make the servlet execution abort when the session requests a time out.`

suggested edit:

`Set this property to false to stop the servlet execution when the session requests a timeout. The default value is true.`